roles/2-common/tasks/packages.yml

```diff
@@ -30,6 +30,8 @@
       #- openssh-server    # ssh (Raspbian) or openssh-server (other OS's) already installed by 1-prep's roles/sshd/tasks/main.yml
       - pandoc             # For /usr/bin/iiab-refresh-wiki-docs
       - pastebinit         # For /usr/bin/iiab-diagnostics
+      #- python3-pip       # Already installed by /opt/iiab/iiab/scripts/ansible -- this auto-installs 'python3-setuptools' and 'python3' etc
+      - python3-venv       # 2021-07-29: Unlike Debian 11, this is required on Ubuntu 20.04 (where packages like python3-pymysql -> python3 don't drag in 'python3-venv') -- for roles like {calibre-web, jupyterhub, lokole} -- whereas roles/kalite uses (virtual) package 'virtualenv' for Python 2
```

Comment:
For use with Ansible module: [pip](https://docs.ansible.com/ansible/latest/modules/pip_module.html) -m venv